toolkits/springer/packages/springer-buttons/scss/10-settings/buttons.scss

```diff
@@ -0,0 +1,78 @@
+/**
```

Comment:
This file should be `button.scss` (now you've renamed global)

Reply:
I noticed some packages have suffix `-override` - is that something to keep consistency with?

Reply:
I started doing that initially, but it's actually an incorrect word for what is happening, as it could be extending rather than overriding.

Reply:
So we opted for the brand to be a suffix. But haven't retrospectively fix the others.